BinariesAPI.md

```diff
@@ -0,0 +1,107 @@
+# JULEP BinariesAPI
+
+- **Title:** A structured HTTP API for getting Julia binaries
+- **Authors:** Eric Davies <<iamed2@gmail.com>>
+- **Created:** June 16, 2017
+- **Status:** proposed
+
+## Abstract
+
+The URLs for fetching Julia binaries of various versions have changed a few times across
```

Comment:
Well I recall the use of `pre` vs `latest` was used for 0.4 pre-releases in 0.4. I'll admit that some of the work on my part was due to lack of knowledge/documentation (I recall parsing the downloads page at one point).